liquibase.exception.LockException: java.lang.ClassCastException: java.time.LocalDateTime cannot be cast to java.util.Date

<!--- This environment context section helps us quickly review your PR. 
      Please take a minute to fill-out this information. -->

## Environment
**Liquibase Version**:

4.3.1

**Liquibase Integration & Version**: <Pick one: CLI, maven, gradle, spring boot, servlet, etc.>

spring boot

**Liquibase Extension(s) & Version**: 

**Database Vendor & Version**:
MySQL  5.7.25

**Operating System Type & Version**:
windows 10

## Description
```
liquibase.exception.LockException: java.lang.ClassCastException: java.time.LocalDateTime cannot be cast to java.util.Date
	at liquibase.lockservice.StandardLockService.listLocks(StandardLockService.java:404)
	at liquibase.lockservice.StandardLockService.waitForLock(StandardLockService.java:225)
	at liquibase.Liquibase.lambda$update$1(Liquibase.java:219)
	at liquibase.Scope.lambda$child$0(Scope.java:160)
	at liquibase.Scope.child(Scope.java:169)
	at liquibase.Scope.child(Scope.java:159)
	at liquibase.Scope.child(Scope.java:138)
	at liquibase.Liquibase.runInScope(Liquibase.java:2322)
	at liquibase.Liquibase.update(Liquibase.java:216)
	at liquibase.Liquibase.update(Liquibase.java:202)
	at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:322)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:275)
	at org.springframework.boot.autoconfigure.liquibase.DataSourceClosingSpringLiquibase.afterPropertiesSet(DataSourceClosingSpringLiquibase.java:46)
	at com.test.jpa.config.liquibase.AsyncSpringLiquibase.initDb(AsyncSpringLiquibase.java:100)
	at com.test.jpa.config.liquibase.AsyncSpringLiquibase.lambda$afterPropertiesSet$0(AsyncSpringLiquibase.java:75)
	at com.test.core.async.ExceptionHandlingAsyncTaskExecutor.lambda$createWrappedRunnable$1(ExceptionHandlingAsyncTaskExecutor.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassCastException: java.time.LocalDateTime cannot be cast to java.util.Date
	at liquibase.lockservice.StandardLockService.listLocks(StandardLockService.java:396)
	... 18 common frames omitted
```

We are using latest 4.3.2, but we have the same issue. Wasn't this supposed to be fixed in 4.3.2?

```
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [org/springframework/boot/autoconfigure/liquibase/LiquibaseAutoConfiguration$LiquibaseConfiguration.class]: Invocation of init method failed; nested exception is liquibase.exception.LockException: java.lang.ClassCastException: class java.time.LocalDateTime cannot be cast to class java.util.Date (java.time.LocalDateTime and java.util.Date are in module java.base of loader 'bootstrap')
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1786)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:602)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:524)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:322)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1154)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:908)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:583)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:769)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:761)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:426)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:326)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1313)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1302)
	at net.iconomi.core.IconomiCoreApplication.main(IconomiCoreApplication.java:39)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:107)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:88)
Caused by: liquibase.exception.LockException: java.lang.ClassCastException: class java.time.LocalDateTime cannot be cast to class java.util.Date (java.time.LocalDateTime and java.util.Date are in module java.base of loader 'bootstrap')
	at liquibase.lockservice.StandardLockService.listLocks(StandardLockService.java:404)
	at liquibase.lockservice.StandardLockService.waitForLock(StandardLockService.java:225)
	at liquibase.Liquibase.lambda$update$1(Liquibase.java:220)
	at liquibase.Scope.lambda$child$0(Scope.java:160)
	at liquibase.Scope.child(Scope.java:169)
	at liquibase.Scope.child(Scope.java:159)
	at liquibase.Scope.child(Scope.java:138)
	at liquibase.Liquibase.runInScope(Liquibase.java:2370)
	at liquibase.Liquibase.update(Liquibase.java:217)
	at liquibase.Liquibase.update(Liquibase.java:203)
	at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:321)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:275)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1845)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1782)
	... 26 common frames omitted
Caused by: java.lang.ClassCastException: class java.time.LocalDateTime cannot be cast to class java.util.Date (java.time.LocalDateTime and java.util.Date are in module java.base of loader 'bootstrap')
	at liquibase.lockservice.StandardLockService.listLocks(StandardLockService.java:396)
	... 39 common frames omitted
```

--- a/liquibase-core/pom.xml
+++ b/liquibase-core/pom.xml
@@ -47,7 +47,14 @@
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
-            <version>3.3.3</version>
+            <version>3.8.0</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-inline</artifactId>
+            <version>3.8.0</version>
             <scope>test</scope>
         </dependency>
 

--- a/liquibase-core/src/main/java/liquibase/lockservice/StandardLockService.java
+++ b/liquibase-core/src/main/java/liquibase/lockservice/StandardLockService.java
@@ -26,6 +26,8 @@ import liquibase.structure.DatabaseObject;
 import liquibase.structure.core.Table;
 
 import java.text.DateFormat;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
 import java.util.*;
 
 import static java.util.ResourceBundle.getBundle;
@@ -387,10 +389,17 @@ public class StandardLockService implements LockService {
                     locked = (Boolean) lockedValue;
                 }
                 if ((locked != null) && locked) {
+                    Object lockGranted = columnMap.get("LOCKGRANTED");
+                    final Date castedLockGranted;
+                    if (lockGranted instanceof LocalDateTime) {
+                        castedLockGranted = Date.from(((LocalDateTime) lockGranted).atZone(ZoneId.systemDefault()).toInstant());
+                    } else {
+                        castedLockGranted = (Date)lockGranted;
+                    }
                     allLocks.add(
                             new DatabaseChangeLogLock(
                                     ((Number) columnMap.get("ID")).intValue(),
-                                    (Date) columnMap.get("LOCKGRANTED"),
+                                    castedLockGranted,
                                     (String) columnMap.get("LOCKEDBY")
                             )
                     );

--- a/liquibase-core/src/test/java/liquibase/lockservice/StandardLockServiceTest.java
+++ b/liquibase-core/src/test/java/liquibase/lockservice/StandardLockServiceTest.java
@@ -1,341 +1,102 @@
 package liquibase.lockservice;
 
+import liquibase.Scope;
+import liquibase.exception.DatabaseException;
+import liquibase.exception.LockException;
+import liquibase.executor.Executor;
+import liquibase.executor.ExecutorService;
+import org.assertj.core.api.Assertions;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.mockito.MockedStatic;
 
-@SuppressWarnings({"EqualsWhichDoesntCheckParameterClass"})
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
 public class StandardLockServiceTest {
 
+    public static final LocalDateTime LOCKGRANTED_DATETIME = LocalDateTime.of(2021, 6, 1, 20, 34);
+    public static final Date LOCKGRANTED_DATE = Date.from(Instant.ofEpochMilli(1622580938000l));
+    public static final int ID0 = 23;
+    public static final int ID1 = 42;
+    public static final String LOCKEDBY = "The devil";
     private StandardLockService lockService;
+    private Scope mockedScope;
 
     @Before
-    public void before() {
-        lockService = new StandardLockService();
-        lockService.reset();
+    public void before() throws Exception {
+        Executor executor = Mockito.mock(Executor.class);
+        Mockito
+            .when(executor.queryForList(Mockito.any()))
+            .thenReturn(sampleLockData());
+        ExecutorService executorService = Mockito.mock(ExecutorService.class);
+        Mockito
+            .when(executorService.getExecutor(Mockito.any(), Mockito.any()))
+            .thenReturn(executor);
+
+        Map<String, Object> scopeValues = new TreeMap<>();
+
+        scopeValues.put(ExecutorService.class.getName(), executorService);
+
+        mockedScope = Mockito.mock(Scope.class);
+        Mockito.when(mockedScope.getSingleton(ExecutorService.class)).thenReturn(executorService);
+
+        lockService = new StandardLockService() {
+            @Override
+            public boolean hasDatabaseChangeLogLockTable() throws DatabaseException {
+                return true;
+            }
+        };
     }
 
-    @After
-    public void after() {
-        lockService.reset();
+    @Test
+    public void shouldAcceptLocaldatetimeInLOCKGRANTED() throws LockException {
+        try (MockedStatic<Scope> scope = Mockito.mockStatic(Scope.class)) {
+            scope.when(Scope::getCurrentScope).thenReturn(mockedScope);
+            DatabaseChangeLogLock[] databaseChangeLogLocks = lockService.listLocks();
+
+            Assertions.assertThat(databaseChangeLogLocks)
+                .hasSize(2);
+
+            Assertions.assertThat(databaseChangeLogLocks[0].getId())
+                .isEqualTo(ID0);
+            Assertions.assertThat(databaseChangeLogLocks[0].getLockGranted())
+                .isEqualTo(Date.from((LOCKGRANTED_DATETIME).atZone(ZoneId.systemDefault()).toInstant()));
+            Assertions.assertThat(databaseChangeLogLocks[0].getLockedBy())
+                .isEqualTo(LOCKEDBY);
+
+            Assertions.assertThat(databaseChangeLogLocks[1].getId())
+                .isEqualTo(ID1);
+            Assertions.assertThat(databaseChangeLogLocks[1].getLockGranted())
+                .isEqualTo(LOCKGRANTED_DATE);
+            Assertions.assertThat(databaseChangeLogLocks[1].getLockedBy())
+                .isEqualTo(LOCKEDBY);
+        }
     }
 
-//    @Test
-//    public void aquireLock_hasLockAlready() throws Exception {
-//        Database database = createMock(Database.class);
-//        replay(database);
-//
-//        lockService.setDatabase(database);
-//        assertFalse(lockService.hasChangeLogLock());
-//
-//        Field field = lockService.getClass().getDeclaredField("hasChangeLogLock");
-//        field.setAccessible(true);
-//        field.set(lockService, true);
-//
-//        assertTrue(lockService.hasChangeLogLock());
-//
-//        assertTrue(lockService.acquireLock());
-//    }
+    private static List<Map<String, ?>> sampleLockData() {
+        Map<String, Object> columnMapRow0 = new TreeMap<>();
 
+        columnMapRow0.put("ID", ID0);
+        columnMapRow0.put("LOCKED", true);
+        columnMapRow0.put("LOCKGRANTED", LOCKGRANTED_DATETIME);
+        columnMapRow0.put("LOCKEDBY", LOCKEDBY);
 
-//    @Test
-//    public void acquireLock_tableExistsNotLocked() throws Exception {
-//        Database database = createMock(Database.class);
-//        Executor executor = createMock(Executor.class);
-//
-////        database.checkDatabaseChangeLogLockTable();
-////        expectLastCall();
-//
-//        database.rollback();
-//        expectLastCall().anyTimes();
-//
-//        database.setCanCacheLiquibaseTableInfo(true);
-//        expectLastCall();
-//
-//        expect(executor.queryForObject(isA(SelectFromDatabaseChangeLogLockStatement.class), eq(Boolean.class))).andReturn(false);
-//
-//        executor.comment("Lock Database");
-//        expectLastCall();
-//
-//        expect(executor.update(isA(LockDatabaseChangeLogStatement.class))).andReturn(1);
-//
-//        database.commit();
-//        expectLastCall();
-//
-//        replay(executor);
-//        replay(database);
-//        Scope.getCurrentScope().getSingleton(ExecutorService.class).setExecutor(database, executor);
-//
-//        LockServiceImpl service = new LockServiceImpl();
-//        service.setDatabase(database);
-//        assertTrue(service.acquireLock());
-//
-//        verify(database);
-//        verify(executor);
-//    }
+        Map<String, Object> columnMapRow1 = new TreeMap<>();
 
-//    @Test
-//    public void acquireLock_tableExistsIsLocked() throws Exception {
-//        Database database = createMock(Database.class);
-//        Executor executor = createMock(Executor.class);
-//
-////        database.checkDatabaseChangeLogLockTable();
-////        expectLastCall();
-//
-//        database.rollback();
-//        expectLastCall().anyTimes();
-//
-//        expect(executor.queryForObject(isA(SelectFromDatabaseChangeLogLockStatement.class), eq(Boolean.class))).andReturn(true);
-//
-//        replay(database);
-//        replay(executor);
-//        Scope.getCurrentScope().getSingleton(ExecutorService.class).setExecutor(database, executor);
-//
-//        LockServiceImpl service = new LockServiceImpl();
-//        service.setDatabase(database);
-//        assertFalse(service.acquireLock());
-//
-//        verify(database);
-//    }
+        columnMapRow1.put("ID", ID1);
+        columnMapRow1.put("LOCKED", 1);
+        columnMapRow1.put("LOCKGRANTED", LOCKGRANTED_DATE);
+        columnMapRow1.put("LOCKEDBY", LOCKEDBY);
 
-//    @Test
-//    public void waitForLock_notLocked() throws Exception {
-//        Database database = createMock(Database.class);
-//        Executor executor = createMock(Executor.class);
-//
-//
-////        database.checkDatabaseChangeLogLockTable();
-////        expectLastCall();
-//
-//        database.setCanCacheLiquibaseTableInfo(true);
-//        expectLastCall();
-//
-//        database.rollback();
-//        expectLastCall().anyTimes();
-//
-//        expect(executor.queryForObject(isA(SelectFromDatabaseChangeLogLockStatement.class), eq(Boolean.class))).andReturn(false);
-//
-//        expect(database.getDatabaseChangeLogLockTableName()).andReturn("DATABASECHANGELOGLOCK");
-//        expect(database.getLiquibaseCatalogName()).andReturn("LQCAT");
-//        expect(database.getLiquibaseSchemaName()).andReturn("LQSCHM");
-//        database.setCanCacheLiquibaseTableInfo(true);
-//        expectLastCall();
-//
-//        executor.comment("Lock Database");
-//        expectLastCall();
-//
-//        expect(executor.update(isA(LockDatabaseChangeLogStatement.class))).andReturn(1);
-//
-//        database.commit();
-//        expectLastCall();
-//
-//        replay(database);
-//        replay(executor);
-//        Scope.getCurrentScope().getSingleton(ExecutorService.class).setExecutor(database, executor);
-//
-//        LockServiceImpl service = new LockServiceImpl();
-//        service.setDatabase(database);
-//        service.waitForLock();
-//
-//        verify(database);
-//        verify(executor);
-//    }
-
-//    @Test
-//    public void waitForLock_lockedThenReleased() throws Exception {
-//        Database database = createMock(Database.class);
-//        Executor executor = createMock(Executor.class);
-//
-////        database.checkDatabaseChangeLogLockTable();
-////        expectLastCall().anyTimes();
-//
-//        database.setCanCacheLiquibaseTableInfo(true);
-//        expectLastCall();
-//
-//        expect(executor.queryForObject(isA(SelectFromDatabaseChangeLogLockStatement.class), eq(Boolean.class))).andReturn(true);
-//        expect(executor.queryForObject(isA(SelectFromDatabaseChangeLogLockStatement.class), eq(Boolean.class))).andReturn(true);
-//        expect(executor.queryForObject(isA(SelectFromDatabaseChangeLogLockStatement.class), eq(Boolean.class))).andReturn(true);
-//        expect(executor.queryForObject(isA(SelectFromDatabaseChangeLogLockStatement.class), eq(Boolean.class))).andReturn(false);
-//
-//        executor.comment("Lock Database");
-//        expectLastCall();
-//
-//        database.rollback();
-//        expectLastCall().anyTimes();
-//
-//        expect(executor.update(isA(LockDatabaseChangeLogStatement.class))).andReturn(1);
-//
-//        database.commit();
-//        expectLastCall();
-//
-//        replay(database);
-//        replay(executor);
-//        Scope.getCurrentScope().getSingleton(ExecutorService.class).setExecutor(database, executor);
-//
-//        LockServiceImpl service = new LockServiceImpl();
-//        service.setDatabase(database);
-//        service.setChangeLogLockRecheckTime(1);
-//        service.waitForLock();
-//
-//        verify(database);
-//        verify(executor);
-//    }
-
-//    @Test
-//    public void waitForLock_lockedThenTimeout() throws Exception {
-//        Database database = createMock(Database.class);
-//        Executor executor = createMock(Executor.class);
-//
-////        database.checkDatabaseChangeLogLockTable();
-////        expectLastCall().anyTimes();
-//
-////        expect(executor.queryForObject(isA(SelectFromDatabaseChangeLogLockStatement.class), eq(Boolean.class))).andReturn(true).anyTimes();
-////        expect(database.hasDatabaseChangeLogLockTable()).andReturn(true);
-//
-//        List<Map<String, ?>> resultList = new ArrayList<Map<String, ?>>();
-//        Map<String, Object> result = new HashMap<String, Object>();
-//        result.put("ID", 1);
-//        result.put("LOCKED", true);
-//        Date lockDate = new Date();
-//        result.put("LOCKGRANTED", lockDate);
-//        result.put("LOCKEDBY", "Locker");
-//        resultList.add(result);
-//
-//        expect(executor.queryForList(isA(SelectFromDatabaseChangeLogLockStatement.class))).andReturn(resultList);
-//
-//        database.rollback();
-//        expectLastCall().anyTimes();
-//
-//        replay(database);
-//        replay(executor);
-//        Scope.getCurrentScope().getSingleton(ExecutorService.class).setExecutor(database, executor);
-//
-//        LockServiceImpl service = new LockServiceImpl();
-//        service.setDatabase(database);
-//        service.setChangeLogLockWaitTime(10);
-//        service.setChangeLogLockRecheckTime(5);
-//
-//        try {
-//            service.waitForLock();
-//            fail("Should have thrown exception");
-//        } catch (LockException e) {
-//            assertEquals("Could not acquire change log lock.  Currently locked by Locker since " + DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT).format(lockDate), e.getMessage());
-//        }
-//
-//        verify(database);
-//    }
-
-//    @Test
-//    public void releaseLock_tableExistsAndLocked() throws Exception {
-//        Database database = createMock(Database.class);
-//        Executor executor = createMock(Executor.class);
-//
-//        expect(executor.update(isA(UnlockDatabaseChangeLogStatement.class))).andReturn(1);
-////        expect(database.hasDatabaseChangeLogLockTable()).andReturn(true);
-//        database.commit();
-//        expectLastCall().atLeastOnce();
-//
-//        database.setCanCacheLiquibaseTableInfo(false);
-//        expectLastCall();
-//
-//        executor.comment("Release Database Lock");
-//        expectLastCall().anyTimes();
-//
-//        database.rollback();
-//        expectLastCall().anyTimes();
-//
-//        replay(database);
-//        replay(executor);
-//        Scope.getCurrentScope().getSingleton(ExecutorService.class).setExecutor(database, executor);
-//
-//        LockServiceImpl service = new LockServiceImpl();
-//        service.setDatabase(database);
-//        service.releaseLock();
-//
-//        verify(database);
-//    }
-
-//    @Test
-//    public void listLocks_hasLocks() throws Exception {
-//        Database database = createMock(Database.class);
-//        Executor executor = createMock(Executor.class);
-//
-////        database.checkDatabaseChangeLogLockTable();
-////        expectLastCall().anyTimes();
-//
-////        expect(executor.queryForObject(isA(SelectFromDatabaseChangeLogLockStatement.class), eq(Boolean.class))).andReturn(true).anyTimes();
-////        expect(database.hasDatabaseChangeLogLockTable()).andReturn(true);
-//
-//        List<Map<String, ?>> resultList = new ArrayList<Map<String, ?>>();
-//        Map<String, Object> result = new HashMap<String, Object>();
-//        result.put("ID", 1);
-//        result.put("LOCKED", true);
-//        Date lockDate = new Date();
-//        result.put("LOCKGRANTED", lockDate);
-//        result.put("LOCKEDBY", "Locker");
-//        resultList.add(result);
-//
-//        expect(executor.queryForList(isA(SelectFromDatabaseChangeLogLockStatement.class))).andReturn(resultList);
-//
-//        replay(database);
-//        replay(executor);
-//        Scope.getCurrentScope().getSingleton(ExecutorService.class).setExecutor(database, executor);
-//
-//        LockServiceImpl service = new LockServiceImpl();
-//        service.setDatabase(database);
-//        DatabaseChangeLogLock[] locks = service.listLocks();
-//        assertEquals(1, locks.length);
-//        assertEquals(1, locks[0].getId());
-//        assertEquals("Locker", locks[0].getLockedBy());
-//        assertEquals(lockDate, locks[0].getLockGranted());
-//
-//        verify(database);
-//    }
-
-//    @Test
-//    public void listLocks_tableExistsUnlocked() throws Exception {
-//        Database database = createMock(Database.class);
-//        Executor executor = createMock(Executor.class);
-//
-////        database.checkDatabaseChangeLogLockTable();
-////        expectLastCall().anyTimes();
-//
-////        expect(executor.queryForObject(isA(SelectFromDatabaseChangeLogLockStatement.class), eq(Boolean.class))).andReturn(true).anyTimes();
-////        expect(database.hasDatabaseChangeLogLockTable()).andReturn(true);
-//
-//        List<Map<String, ?>> resultList = new ArrayList<Map<String, ?>>();
-//
-//        expect(executor.queryForList(isA(SelectFromDatabaseChangeLogLockStatement.class))).andReturn(resultList);
-//
-//        replay(database);
-//        replay(executor);
-//        Scope.getCurrentScope().getSingleton(ExecutorService.class).setExecutor(database, executor);
-//
-//        LockServiceImpl service = new LockServiceImpl();
-//        service.setDatabase(database);
-//        DatabaseChangeLogLock[] locks = service.listLocks();
-//        assertEquals(0, locks.length);
-//
-//        verify(database);
-//    }
-
-//    @Test
-//    public void listLocks_tableDoesNotExists() throws Exception {
-//        Database database = createMock(Database.class);
-//
-////        expect(database.hasDatabaseChangeLogLockTable()).andReturn(false);
-//
-//        replay(database);
-//
-//        LockServiceImpl service = new LockServiceImpl();
-//        service.setDatabase(database);
-//        DatabaseChangeLogLock[] locks = service.listLocks();
-//        assertEquals(0, locks.length);
-//
-//        verify(database);
-//    }
-
-    public void testNothing() {
-
+        return Arrays.asList(columnMapRow0, columnMapRow1);
     }
-
 }

